backend/src/main/java/gov/cdc/usds/simplereport/api/pxp/PatientLinkMutationResolver.java

```diff
@@ -1,21 +1,21 @@
 package gov.cdc.usds.simplereport.api.pxp;
 
-import gov.cdc.usds.simplereport.service.PatientLinkService;
+import gov.cdc.usds.simplereport.service.TestResultsDeliveryService;
 import gov.cdc.usds.simplereport.service.model.SmsAPICallResult;
 import gov.cdc.usds.simplereport.service.sms.SmsService;
 import graphql.kickstart.tools.GraphQLMutationResolver;
 import java.util.List;
 import java.util.UUID;
-import org.springframework.beans.factory.annotation.Autowired;
+import lombok.RequiredArgsConstructor;
```

Comment:
Another one bites the dust 🪓 thanks for consistently keeping this improvement in mind